www/_posts/2018-09-27-cordova-cli-8.1.1.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+author:
+    name: Chris Brody
+    url: https://twitter.com/brodybits
+title:  "Cordova-CLI Hotfix Release"
+categories: news
+tags: release tools
+---
+
+We are happy to announce that Cordova CLI `8.1.1` hotfix was released in September 2018.
+
+## Release Highlights
+
+The purpose of this hotfix release is to resolve a couple items that were broken in the Cordova CLI `8.1.0` release:
+
+* [GH-339](https://github.com/apache/cordova-cli/issues/339) Fix bin/cordova on Node.js 4
```

Comment:
links are to issues, but text seems to be from PRs/commits?

Reply:
Yes. There was one PR with 2 commits to resolve the 2 issues. I can resolve this in one of two ways:
1. Change GH numbers to the PR number and change above text to be more consistent with this solution
2. Change the text to reflect the issue actually solved

I would probably favor the first one. I think it is better for the release note to tell what was actually changed, especially in a hot fix.